docs/development/authentication.md

```diff
@@ -42,6 +42,23 @@ details worth understanding:
   password in the development environment, those passwords won't
   work.  It also prints out the user's **current** API key.
 
+### Apple
+
+* Visit https://developer.apple.com/account/resources/,
+  Enable App ID and Create a Services ID with the instructions in
+  https://help.apple.com/developer-account/?lang=en#/dev1c0e25352 .
+  When prompted for a "Return URL", enter
+  `http://zulipdev.com:9991/complete/apple/` .
+
+* [Create a Sign in with Apple private key](https://help.apple.com/developer-account/?lang=en#/dev77c875b7e)
+
+* In `dev-secrets.conf`, set `social_auth_apple_client` to your
+  "Services ID" (eg. com.application.your),
```

Comment:
Shouldn't this be `App ID`, not `Services ID`? At least that's what seems right based on @chrisbobbe testing